styles.css

```diff
@@ -3,15 +3,15 @@
   color: #FFF;
 }
 
-@font-face {
-    font-family: Monttserrat;
```

Comment:
Curious as to why this global font-face selector wasn't the preferred method? Glad you found a more concise way of implementing the font-family, though.